docs/content/guides/developer/sui-101.mdx

```diff
@@ -12,18 +12,26 @@ Everything on the Sui blockchain is an object. These topics use code examples to
 - [Create Coins](./sui-101/create-coin.mdx)
 - [Create NFTs](./sui-101/create-nft.mdx)
 
-## Building programmable transaction blocks
+## Working with programmable transaction blocks
```

Comment:
nit: in some places we capitalize the words in titles, in others we don't.

Reply:
The only headings that _should_ be title case are ones that are the exact title of a page. This one's tricky as the page is titled Working with PTBs. Didn't want to use that because the acronym in the ToC on the right might be confusing. But it's defined in the first sentence so using Working with PTBs as the title.